cmake/templates/pkgConfig.cmake.in

```diff
@@ -78,6 +78,15 @@ else()
   set(@PROJECT_NAME@_PREFIX ${@PROJECT_NAME@_INSTALL_PREFIX})
 endif()
 
+if(CMAKE_CROSSCOMPILING)
+  if("@DEVELSPACE@" STREQUAL "TRUE")
+    # can not disable host filesystem altogether; otherwise files in src/devel are no longer found
+    set(FIND_XXX_LOCATION_OPTION CMAKE_FIND_ROOT_PATH_BOTH)
```

Comment:
The name `FIND_XXX_LOCATION_OPTION` makes it look like an "official" variable of CMake. Since it is only a custom local variable please use `_find_root_path_both` instead.